front/app/components/LandingPageComponents/admin/CTARadioButtons.tsx

```diff
@@ -0,0 +1,108 @@
+import { Box, Input, Label, Radio } from '@citizenlab/cl2-component-library';
```

Comment:
This is a refactor of SettingsRadioButtons that got deleted.

-Simpler code, less deep component tree
-Also added Error components to display API errors
-Added type checking to the different labelMessages to avoid silent bugs
-Moved this to the core since it's now used by custom pages (in core) + customizable homepage banner (in module).